src/umpire/strategy/DynamicPool.hpp

```diff
@@ -68,7 +68,7 @@ class DynamicPool :
         Allocator allocator,
         const std::size_t min_initial_alloc_size = (512 * 1024 * 1024),
         const std::size_t min_alloc_size = (1 * 1024 *1024),
-        Coalesce_Heuristic coalesce_heuristic = &heuristic_all_allocations_are_releaseable) noexcept;
+        Coalesce_Heuristic coalesce_heuristic = heuristic_percent_releasable(0)) noexcept;
```

Comment:
When we discussed this previously, I thought you expressed an interest in making the default be that the heuristic was disabled which is what this effectively does.  I was concerned about making this the default in 0.3.1 because I didn't believe the 0.3.1 heuristic framework offered an adequate interface to provide other options and disabling the heuristic would be different behavior from the previous release.

My preference is to make the default be heuristic_percent_releasable(100) which is what 0.3.0 and 0.3.1 provide.  If you are okay with that, I will switch to that.

Reply:
Yup, I remember 😄 we can make the default have the same behavior as 0.3.0 and 0.3.1

Reply:
Please mark this resolved once you make the switch.

Reply:
Will do.  Thanks